main/zoe/api/zoe.md

```diff
@@ -0,0 +1,77 @@
+# Zoe
+
+Zoe is a long-lived and well-trusted contract that enforces offer safety for the contracts that use it. Zoe has a single `seatAssay` and a single `escrowReceiptAssay` for the entirety of its lifetime. By having a reference to Zoe, a user can get the `seatAssay` or `escrowReceiptAssay` and thus validate any `seat` or `escrowReceipt` they receive from someone else.
```

Comment:
This should be `inviteAssay` rather than `seatAssay` and `invite` rather than `seat`.

Reply:
Just saw that it's this way in the zoe.chainmail. Sorry about that. I will update there and make notes here.

Reply:
udpated